packagedb/models.py

```diff
@@ -81,6 +87,105 @@ def paginated(self, per_page=5000):
             page = paginator.page(page_number)
             yield from page.object_list
 
+    # Based on class PurlValidateResponseSerializer(Serializer).  Added here because when added to serializers.py and imported raises a circular import error.
```

Comment:
Does this mean that the circular import happens when you add this code to serializers.py and then try to import it from somewhere like views.py?

Reply:
@JonoYang   Glad you asked about the custom serializer I created inside `models.py`.  

This means that when I initially defined that new serializer in `packagedb/serializers.py` and tried to import it into `models.py`, where I am using the local one now, I got the circular import error.  Maybe I don't need a serializer here?  

I modeled my approach on what we do in the `api.py` `PurlValidateViewSet()` class with the `PurlValidateResponseSerializer` we import from `packagedb.serializers`.  Essentially I'm trying to mimic in my purl-validate UI the dictionary that our validate endpoint returns -- 4 fields, and I added one or two more for my UI validation messaging (preventing an exception from being raised and interrupting the process).  `packagedb/serializers.py` has numerous imports from `packagedb/models`, so that's the source of the circular import issue.